src/sentry/tasks/store.py

```diff
@@ -293,6 +293,11 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None, **kwar
     try:
         manager = EventManager(data)
         manager.save(project)
+    except HashDiscarded as exc:
+        error_logger.info('discarded.hash', extra={
```

Comment:
I don't think this is an error (its logging to `sentry.errors.events`), it's sort of expected to happen